libs/langchain/langchain/llms/google_palm.py

```diff
@@ -1,7 +1,5 @@
 from langchain_community.llms.google_palm import (
     GooglePalm,
-    _strip_erroneous_leading_spaces,
-    completion_with_retry,
 )
 
-__all__ = ["completion_with_retry", "_strip_erroneous_leading_spaces", "GooglePalm"]
```

Comment:
shouldn't change this for now, we'll remove all these re-exports in a single pr (to consolidate everything into one breaking change)